src/helpers.ts

```diff
@@ -345,3 +345,13 @@ export async function isPullRequestTested(octokit: OctokitType, pullRequest: Pul
 
     return testFilePaths.length > 0;
 };
+
+/**
+ * Fails if the pull request does not have any description.
+ */
+export async function ensurePullRequestHasSomeDescription(octokit: OctokitType, pullRequest: PullRequest) {
+    console.log('Pull request description', pullRequest.body);
+    if (!pullRequest.body) {
+        await fail(pullRequest, 'Pull request must have non-empty description!', octokit);
```

Comment:
I'd also describe here what could be there:

> Please explain what you've done, why you've done it this way, and how to test the feature, including anything the reviewer should focus on.

Reply:
yeah, this is not finished, because it doesn't work for some reason :D (the action still passes). I need to debug that. Do you maybe have any idea what can be wrong here? Thx